src/libs/migrations/RenameActiveClientsKey.js

```diff
@@ -0,0 +1,35 @@
+import Onyx from 'react-native-onyx';
+import _ from 'underscore';
+import ONYXKEYS from '../../ONYXKEYS';
+
+// This migration changes the name of the Onyx key ACTIVE_CLIENTS from activeClients2 to activeClients
+export default function () {
+    return new Promise((resolve) => {
+        // Connect to the old key in Onyx to get the old value of activeClients2
+        // then set the new key activeClients to hold the old data
+        // finally remove the old key by setting the value to null
+        const connectionID = Onyx.connect({
+            key: 'activeClients2',
+
+            // Use _.once() here so that it is not triggered again with the multiSet() below
+            callback: _.once((oldActiveClients) => {
+                Onyx.disconnect(connectionID);
+
+                // Fail early here because there is nothing to migrate
+                if (_.isEmpty(oldActiveClients)) {
+                    console.debug('[Migrate Onyx] Skipped migration RenameActiveClientsKey');
+                    return resolve();
+                }
+
+                Onyx.multiSet({
+                    activeClients2: null,
```

Comment:
This doesn't seem to delete the key from `localStorage`. Is there any reason we should not call `AsyncStorage.removeItem()` here?

![Screen Shot 2021-03-08 at 9 14 53 AM](https://user-images.githubusercontent.com/32969087/110369639-c50f7280-7fee-11eb-80af-daf6247c0858.png)

Reply:
Yeah, I mentioned this in the PR description (no worries if you missed it). This is a known limitation. I didn't want to add a `remove()` method to Onyx, `AsyncStorage.removeItem()` shouldn't be called directly, it will get cleared out the next time they are signed out, and it doesn't take up any space.

Reply:
Cool. Yep, you got me I didn't read the description at all 😅 

Just to clarify, did you not want to add a `Onyx.remove()` method because you feel it's incorrect or because we would need to make the change to the Onyx lib?

Reply:
More because I think it's incorrect and there is no need for it. I also don't want to add `Onyx.remove()` because it won't be used in any other code, and I think it adds temptation for it to be used elsewhere in code and could be implemented out of laziness. It's basically the same reason why we removed `Onyx.get()` from the Onyx API. Even though there were valid use cases, there was more of a temptation for it to lead to harmful code than it helped.

Reply:
Ok, thanks for the explanation.